tests/components/template/test_fan.py

```diff
@@ -414,8 +414,9 @@ async def test_invalid_availability_template_keeps_component_available(hass, cap
     await hass.async_block_till_done()
 
     assert hass.states.get("fan.test_fan").state != STATE_UNAVAILABLE
-    assert ("Could not render availability_template template") in caplog.text
-    assert ("UndefinedError: 'x' is undefined") in caplog.text
+
+    assert "TemplateError" in caplog.text
+    assert "x" in caplog.text
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6042836)